src/xpk/core/kueue_manager.py

```diff
@@ -104,7 +105,10 @@ def install_or_upgrade(
     return_code, installed_version = self.get_installed_kueue_version()
 
     if return_code == 0:
-      if installed_version and installed_version > self.kueue_version:
+      if (
+          installed_version
+          and semver_compare(installed_version, self.kueue_version) >= 0
```

Comment:
It used to be strictly greater on purpose. The idea is that if someone does a re-run we'll recreate all the resources.